### ISSUE TITLE
Adrenal hemorrhage

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11098]
**Preferred term label:**
Adrenal hemorrhage

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0151693

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Adrenal hemorrhage

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001280
- label: Adrenal hemorrhage
- parents: [Abnormal adrenal morphology (HP:0011732)](https://hpo.jax.org/app/browse/term/HP:0011732)
- pmid: PMID:38432745|PMID:36404284
- def: Adrenal hemorrhage is bleeding within the capsule of the adrenal gland.
- comment: It is challenging to diagnose because it is considered rare and therefore overlooked in the differential diagnosis, while no clinical features or routine laboratory tests can immediately alert to the diagnosis. Most adrenal hemorrhages were unilateral, presenting incidentally or during trauma evaluation, and occurred secondary to postoperative complication or trauma.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11098
- synonyms:
